app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -10,6 +10,15 @@ module TwoFactorAuthenticatable
   DIRECT_OTP_VALID_FOR_MINUTES = IdentityConfig.store.otp_valid_for
   DIRECT_OTP_VALID_FOR_SECONDS = DIRECT_OTP_VALID_FOR_MINUTES * 60
   REMEMBER_2FA_COOKIE = 'remember_tfa'.freeze
+  AUTH_METHOD_BACKUP_CODE = 'backup_code'
+  AUTH_METHOD_PERSONAL_KEY = 'personal_key'
+  AUTH_METHOD_PIV_CAC = 'piv_cac'
+  AUTH_METHOD_REMEMBER_DEVICE = 'remember_device'
+  AUTH_METHOD_SMS = 'sms'
+  AUTH_METHOD_TOTP = 'totp'
+  AUTH_METHOD_VOICE = 'voice'
+  AUTH_METHOD_WEBAUTHN = 'webauthn'
+  AUTH_METHOD_WEBAUTHN_PLATFORM = 'webauthn_platform'
```

Comment:
if we wanted, we could make a module to namespace these? and then if we ever wanted to treat them like a collection, we could add methods on the module

```suggestion
  module AuthMethod
    BACKUP_CODE = 'backup_code'
    PERSONAL_KEY = 'personal_key'
    PIV_CAC = 'piv_cac'
    REMEMBER_DEVICE = 'remember_device'
    SMS = 'sms'
    TOTP = 'totp'
    VOICE = 'voice'
    WEBAUTHN = 'webauthn'
    PLATFORM = 'webauthn_platform'
  end
```